admin.go

```diff
@@ -0,0 +1,376 @@
+package sarama
+
+import "errors"
+
+type ClusterAdmin interface {
+	// Creates a new topic. This operation is supported by brokers with version 0.10.1.0 or higher.
+	// It may take several seconds after CreateTopic returns success for all the brokers
+	// to become aware that the topic has been created. During this time, ClusterAdmin#listTopics
+	// and ClusterAdmin#describeTopics may not return information about the new topic.
+	// This operation is supported by brokers with version 0.10.1.0 or higher.
+	// The validateOnly option is supported from version 0.10.2.0.
+	CreateTopic(topic string, detail *TopicDetail, validateOnly bool) error
+
+	// Delete a batch of topics.
```

Comment:
This sentence needs to be singular still since the method only deletes one at a time.